tensorflow-cpu.yaml

```diff
@@ -0,0 +1,126 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: tensorflow-cpu
+  version: 0_git20250108
+  epoch: 0
+  description: tensorflow-cpu
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    - x86_64
+  dependencies:
+    runtime:
+      - bash
+      - build-base
+      - curl
+      - pkgconf
+      - py${{vars.py-version}}-pip
+      - python-${{vars.py-version}}-dev
+      - unzip
+
+vars:
+  py-version: 3.11
```

Comment:
I'm sure there's a reason, but why do we pin to 3.11?

Reply:
Also, it seems that 3.12 is not officially supported in their build setups (for which the [issue](https://github.com/tensorflow/build/issues/210)), even though I'm not sure it's an hard requirement.

Reply:
I just added a check that will fail the build when we're using a different version from the default one supported by upstream, referencing their Dockerfile's ARG.

Reply:
Awesome!